src/braket/circuits/gates.py

```diff
@@ -43,14 +43,18 @@ class H(Gate):
     """Hadamard gate."""
 
     def __init__(self):
-        super().__init__(qubit_count=1, ascii_symbols=["H"])
+        super().__init__(qubit_count=self.fixed_qubit_count(), ascii_symbols=["H"])
```

Comment:
Why is this different from all the other gates? is `self.fixed_qubit_count()` or `None` the correct choice here?

Reply:
Oops, missed an undo. Should be None, feel free to suggest and commit that change.

Reply:
Probably faster just to ask you - what happens when qubit_count is `None`? Does it automatically get redirected to `fixed_qubit_count`?

Reply:
So it's defined in `QuantumOperation`: if `fixed_qubit_count` is defined, then the argument `qubit_count` is ignored altogether, hence `None`. I would e like to move the argument to the end and set `None` as default, but that would be breaking.